examples/cors/backend/front-envoy.yaml

```diff
@@ -23,8 +23,10 @@ static_resources:
               domains:
               - "*"
               cors:
-                allow_origin:
-                - "*"
+                allow_origin_string_match:
+                - safe_regex:
+                    google_re2: {}
+                    regex: .*
```

Comment:
nit: do an exact match on "*" here. Same question though on how this passed tests before?